test/playground/playground.test.ts

```diff
@@ -10,8 +10,8 @@ import { createMock } from 'ts-auto-mock';
 
 it('should work', () => {
   interface A {
-    a: string;
-  }
+  a: string;
```

Comment:
let's not touch this file as much as possible

Reply:
done :)